tests/functional/tnf/test_tnf_deployment.py

```
@@ -0,0 +1,824 @@
+@pytest.fixture(scope="session")
+def tnf_deployment():
+    """
+    Fixture to provide TNF deployment instance
+    """
+    return TNF()
+
+
+@turquoise_squad
+@deployment
+@polarion_id(get_polarion_id())
+def test_tnf_deployment(pvc_factory, pod_factory):
```

Ruff (0.16.1)

[warning] Too many statements (56 > 50)

(PLR0915)